### PR TITLE
(360) Contacts can be added to projects

### PR DESCRIPTION
**Pull request by jacksonj04**
This work introduces the concept of a Contact to a Project, a contact stores the information that relates to a person playing a part in the conversion of a school.

We collect basic information:

- title
- name
- email
- telephone number

We expect to assoicate each Contact to a entity such as a School, Trust, Local authority, Solicitors, etc. but this work can follow later to allow us to think about where we might expose the contacts in the application.

As we have not production data right now this is acceptable, work is planned to introduce this association to contacts very soon.

Note: this is @jacksonj04 work, he is now on Holiday so @mec has picked this up!

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [x] Update the `CHANGELOG.md` if needed.

## Screenshots
image

image

image

image

### CODE REVIEW

**Review by jkempster34 — APPROVED** [review]
This all looks great to me! 🔥 
 
We could perhaps put some thought into the validation messages and page design now, so that we don't need to come back to them at a later date?

Nick also needs to configure his editor to insert new lines at the end of files!

File: app/models/project.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class Project < ApplicationRecord
   has_many :sections, dependent: :destroy
   has_many :notes
+  has_many :contacts, dependent: :destroy
```
[the remark this answers is not in the document]
Comment (mec) [reply app/models/project.rb:4]: For sure! Add a card! 👍

**Comment by mec** [comment]
@jkempster34 

| We could perhaps put some thought into the validation messages and page design now, so that we don't need to come back to them at a later date?

If you have views, share them -> happy to spend a little time here as needed?

**Comment by jkempster34** [comment]
> @jkempster34
> 
> > We could perhaps put some thought into the validation messages and page design now, so that we don't need to come back to them at a later date?
> 
> If you have views, share them -> happy to spend a little time here as needed?

Just making the validations GOV.UK style, so:

- Name
  - blank: Enter a name
- Email
  - not_an_email: Enter an email address in the correct format, like name@example.com
- Title:
  - blank: Enter a title

**Comment by mec** [comment]
@jkempster34 I've popped those validation messages on the end - didn't feel like it needed rebasing.

**Pull request merged**